biomage/account/account.py

```diff
@@ -267,6 +277,87 @@ def create_users_list(user_list, header, input_env, aws_profile, overwrite):
             out.write("%s,%s,%s\n" % (full_name, email, password))
 
 
+@click.command()
+@click.option(
+    "--user_list",
+    required=True,
+    help="User list containing user and email for the new accounts in csv.",
+)
+@click.option(
+    "--experiment_name",
+    required=True,
+    help="Experiment name to be created",
+)
+@click.option(
+    "--samples_path",
+    required=True,
+    help="Local path to the samples for upload",
+)
+@click.option(
+    "--instance_url",
+    required=False,
+    default="https://api-default.scp-staging.biomage.net/",
+    help="URL of the cellenics api",
+)
+@click.option(
+    "-p",
+    "--aws_profile",
+    required=False,
+    default=DEFAULT_AWS_PROFILE,
+    show_default=True,
+    help="The name of the profile stored in ~/.aws/credentials to use.",
+)
+@click.option(
+    "--admin_email",
+    required=True,
+    help="admin email for cognito",
+)
+@click.option(
+    "--admin_password",
+    required=True,
+    help="admin password for cognito",
+)
+def create_process_experiment_list(
+    experiment_name,
+    user_list,
+    samples_path,
+    instance_url,
+    aws_profile,
+    admin_email,
+    admin_password,
+):
+    """
+    Creates users, using the user_list file.
+    Creates experiment, uploads samples and processes the projet
+    for each row in the user_list file.
+    The file should be in csv format.
+    The first column should be the full_name in the format: first_name last_name
+    The second column should be the email.
+    E.g.: Arthur Dent, arthur_dent@galaxy.gl
+    """
+
+    cognito_pool = COGNITO_STAGING_POOL
```

Comment:
Why pass this as an environment variable as opposed to using it as a parameter like we usually do?

Reply:
That's how it's done for the create user accounts I just reuse it